src/core/thread/dua_manager.cpp

```diff
@@ -528,14 +528,16 @@ void DuaManager::PerformNextRegistration(void)
         Get<DataPollSender>().SendFastPolls();
     }
 
+    otLogInfoDua("Sent DUA.req for DUA %s", dua.ToString().AsCString());
+
 exit:
     if (error == kErrorNoBufs)
     {
         UpdateCheckDelay(Mle::kNoBufDelay);
     }
 
+    otLogInfoDua("DuaManager::PerformNextRegistration(): %s", ErrorToString(error));
```

Comment:
```suggestion
    otLogInfoDua("PerformNextRegistration: %s", ErrorToString(error));
```
nit: should already contain enough information.